scripts/functions.sh

```diff
@@ -296,7 +296,32 @@ function follow() {
 function copyOutput() {
   echo "Copying collector output files..."
   mkdir -p ./output
+  # output dir may already include files from previous runs
+  oldFlowFile=$(find ./output -name "*txt" | sort | tail -1)
   ${K8S_CLI_BIN} cp -n "$namespace" collector:output ./output
+  flowFile=$(find ./output -name "*txt" | sort | tail -1)
+  if [[ -n "$flowFile" && "$oldFlowFile" != "$flowFile" ]] ; then
+    buildJSON "$flowFile"
+    rm $flowFile
+  fi
+}
+
+function buildJSON() {
+  file=$1
+  filename=$(basename "$file")
+  TMP_FILE="/$MANIFEST_OUTPUT_PATH/$filename"
+  cp "$file" "$TMP_FILE"
+  filenamePrefix=$(echo "$filename" | sed -E 's/(.*)\..*/\1/')
+  UPDATED_JSON_FILE="/$MANIFEST_OUTPUT_PATH/$filenamePrefix.json"
+  { 
+    echo "["
+    # remove last line and "," (last character) of the last flowlog for valid json
+    sed '$d' "$TMP_FILE" | sed '$ s/.$//'
+    echo "]"
+  } >> "$UPDATED_JSON_FILE"
+  dirpath=$(dirname "$file")
+  mv "$UPDATED_JSON_FILE" "$dirpath"
+  rm "$TMP_FILE"
```

Comment:
Now that we have a txt file as input and a json file as output, we don't need to rely on a `TMP_FILE` anymore.
The json file can be written in the final directory as it's done all at once using the brackets + redirect

We sould be able to simplify all of this by:
```suggestion
  dirpath=$(dirname "$file")
  filenamePrefix=$(echo "$filename" | sed -E 's/(.*)\..*/\1/')
  { 
    echo "["
    # remove last line and "," (last character) of the last flowlog for valid json
    sed '$d' "$file" | sed '$ s/.$//'
    echo "]"
  } >> "$dirpath/$filenamePrefix.json"
```

Reply:
What about the format suggested in https://github.com/netobserv/network-observability-cli/pull/201#discussion_r1977114600 ?

Reply:
I just tried with yq, it takes about 30 seconds to return: 
```
$ time yq --inplace -p=json -o=json '.' /tmp/2025-03-03T180301Z1.json
yq --inplace -p=json -o=json '.' /tmp/2025-03-03T180301Z1.json  34.34s user 2.23s system 165% cpu 22.116 total
```

vs jq:

```
$ time jq empty < /tmp/2025-03-03T180301Z1.json
jq empty < /tmp/2025-03-03T180301Z1.json  2.11s user 0.25s system 97% cpu 2.421 total
```

I am hesitant to use `yq` way to validate json given the high processing time,  we can probably rely on `jq` since it's widely prevalent in usage than `yq` . If `jq` is absent we can simply skip validation and assume it'd be valid json, wdyt?

Reply:
Interesting :eyes:  Indeed `yq` seems to be 3 times slower than `jq` on my machine.

I'm not in favor to set `jq` as dependency and we may have issues depending on the `jq` version installed on the machine so I would be in favor to simply remove the format part to keep things simple.

The user can still run the format by himself

Reply:
@jpinsonneau I am okay to remove that part where we test format.  We remove the txt file regardless, correct?

Reply:
yeah keeping only the json non formated is good enough I feel